HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4301,4 +4301,48 @@ def self.get_heatpump_temp_assumptions(heat_pump)
     end
     return hp_min_temp, supp_max_temp
   end
+
+  def self.get_default_duct_surface_area(duct_type, cfa, ncfl, total_cond_flr_area_served, cond_flr_area_served, num_returns)
+    if duct_type == HPXML::DuctTypeSupply
+      if ncfl == 1
+        return 0.27 * cfa * 1.0 * (cond_flr_area_served / total_cond_flr_area_served)
```

Comment:
I'll put it on hold until we make a decision.

Reply:
Note that there are two situations that can encounter this:

A. Only ducted systems
2000ft2 CFA house
1. Central AC - 400ft2 CFA served
2. Central AC - 500ft2 CFA served

B. Ducted and ductless systems:
2000ft2 CFA house
1. Room AC
2. Central AC - 400ft2 CFA served
2. Central AC - 500ft2 CFA served

In situation B, it's technically not possible to know if the sum of CFA served is incorrect. So we will not bother comparing sum of CFA served to CFA.

However, we should go with approach #3: Use the distribution system's conditioned floor area served directly in the equation instead of CFA. So the equation becomes, e.g.:
`        return 0.27 * cond_flr_area_served * 1.0`

And I suggest A) calling out `f_out` as a variable, and B) returning two areas (inside/outside or total/outside).